zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -1528,6 +1528,49 @@ core.api.view.zapHomePath = Gets the path to ZAP's home directory.
 
 core.api.depreciated.alert = Use the API endpoint with the same name in the 'alert' component instead.
 
+custompages.content.type.url=URL
+custompages.content.type.response=Response
+
+custompages.desc=Custom Pages Definition
+
+custompages.dialog.add.title=Add Custom Page
+custompages.dialog.add.button.confirm=Confirm
+custompages.dialog.add.field.label.type=Type:
+custompages.dialog.add.field.label.enabled=Enabled:
+custompages.dialog.add.field.label.regex=Is Regex?
+custompages.dialog.add.field.label.content=Content:
+custompages.dialog.add.field.label.contenttype=Content Type:
```

Comment:
This could also be changed to "location" (there are some variables that still need to be renamed accordingly).